src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/move/JourneyQueryRepository.kt

```diff
@@ -90,19 +90,20 @@ class JourneyQueryRepository(@Autowired val jdbcTemplate: JdbcTemplate) {
                    j.to_nomis_agency_id                                       as journey_to_nomis_agency_id,
                    jtl.site_name                                              as journey_to_site_name,
                    jtl.location_type                                          as journey_to_location_type,
-                   p.price_in_pence                                           as unit_price_in_pence,
+                   COALESCE(pe.price_in_pence, p.price_in_pence)              as unit_price_in_pence,
```

Comment:
Essentially we are saying in these queries a price exception takes precedence if there is one.